bin/oneoff/wipe_data/remove_unused_levelbuilder_users

```diff
@@ -79,12 +79,15 @@ USERS_TO_DELETE = %w(
   896
 ).map(&:to_i).freeze
 
-ActiveRecord::Base.transaction do
-  # ActiveRecord model destroy will raise an error if it encounters an error destroying a single item in the
-  # array and will not continue destroying the remaining items in the input array.
-  # If needed, an individual User can be restored by id: User.with_deleted.find(id).undestroy
-  User.destroy(USERS_TO_DELETE)
+# ActiveRecord `destroy` can take an array argument, but carrying out all destroys in a single database transaction
+# caused the entire operation to be rolledback when an error was raised destroying a single malformed User.
+USERS_TO_DELETE.each do |user_id|
+  User.destroy(user_id)
 
-  # This script is a dry-run unless we comment out this last line
+  # This script is a dry-run unless we comment out this line.
   raise ActiveRecord::Rollback.new, "Intentional rollback"
```

Comment:
This is no longer true - it looks like we've removed the transaction block (or at least this `raise` is now outside of it) so I suspect this script is no longer a dry-run.  Can we put back the transaction, but inside the loop?

Reply:
Oops.  Yes!

Reply:
Done.